pygmt/x2sys.py

```diff
@@ -93,24 +92,26 @@ def x2sys_init(tag, **kwargs):
         The unique name of this data type x2sys TAG.
 
     fmtfile : str
-        Format definition file prefix for this data set [See Format Definition
-        Files for more information]. Specify full path if the file is not in
-        the current directory.
+        Format definition file prefix for this data set (see
+        :gmt-docs:`GMT's Format Definition Files
+        <supplements/x2sys/x2sys_init.html#format-definition-files>`
+        for more information). Specify full path if the file is not in the
+        current directory.
 
         Some file formats already have definition files premade. These include:
 
-        - mgd77 (for plain ASCII MGD77 data files)
-        - mgd77+ (for enhanced MGD77+ netCDF files)
-        - gmt (for old mgg supplement binary files)
-        - xy (for plain ASCII x, y tables)
-        - xyz (same, with one z-column)
-        - geo (for plain ASCII longitude, latitude files)
-        - geoz (same, with one z-column).
+        - **mgd77** (for plain ASCII MGD77 data files)
+        - **mgd77+** (for enhanced MGD77+ netCDF files)
+        - **gmt** (for old mgg supplement binary files)
+        - **xy** (for plain ASCII x, y tables)
+        - **xyz** (same, with one z-column)
+        - **geo** (for plain ASCII longitude, latitude files)
+        - **geoz** (same, with one z-column).
```

Comment:
I have never used this module, so I am not sure whether these should be considered arguments or not (i.e., whether they should be bolded).